tools/repl.md

```diff
@@ -35,6 +35,15 @@ undefined
 undefined
 ```
 
+## Special functions
+
+The REPL provides a couple of special functions, that are always available:
+
+| Function | Description                       |
+| -------- | --------------------------------- |
+| clear()  | Clears the entire terminal screen |
+| close()  | Close the current REPL session    |
```

Comment:
`close()` is on `window` though?

Reply:
Good point, but I think it's also worth mentioning here